joomla/joomla-cms
# [4.2] Replace deprecated method getDbo() by getDatabase()

Pull Request for Issue # .

### Summary of Changes
Simple find and replace PR to replace deprecated method `getDbo()` by `getDatabase()` in components models.

### Testing Instructions
1. Code review
2. Or apply patch, access to random pages from frontend and backend, make sure nothing is broken.
